frontends/web/src/locales/en/app.json

```diff
@@ -1751,7 +1751,8 @@
       "allSessions": "All sessions",
       "disclaimer": "Walletconnect is a protocol to connect to Ethereum based Dapps. These dapps are run by third-party services, so only connect to dapps you trust and be sure to always know what you are signing when making a transaction.",
       "newConnection": "New connection",
-      "noConnectedSessions": "No accounts are currently connected to any dapps."
+      "noConnectedSessions": "No accounts are currently connected to any dapps.",
+      "unspecifiedAccount": "Unspecified account"
```

Comment:
Please update the commit message with what the problem was, what this change does or why continue with "Unspecified account" instead of aborting.